cmd/api/src/api/constant.go

```diff
@@ -59,4 +59,6 @@ const (
 	URIPathVariableTokenID                           = "token_id"
 	URIPathVariableUserID                            = "user_id"
 	URIPathVariableSavedQueryID                      = "saved_query_id"
+	URIPathVariableOIDCProviderID                    = "oidc_provider_id"
```

Comment:
This is from a previous iteration and not needed anymore right?
```suggestion
```

Reply:
Yup! Unused, removing